12-connectivity.tf

```
@@ -39,7 +39,7 @@ resource "azurerm_subnet" "iaas_subnet" {
 
   resource_group_name                            = var.resource_group_name
   virtual_network_name                           = azurerm_virtual_network.virtual_network.name
-  service_endpoints                              = var.iaas_subnet_service_endpoints
+  service_endpoints                              = flatten([var.iaas_subnet_service_endpoints])
```

not sure why you are flattening this?
```suggestion
  service_endpoints                              = var.iaas_subnet_service_endpoints
```

---

Initially tried as a list of strings but TF appears to have a parsing issue as shown here - https://dev.azure.com/hmcts/DevOps/_build/results?buildId=99417&view=logs&j=3d4d5f0d-151b-5e82-4cb8-faa16997da6e&t=e317ea8e-0642-59a1-a5b9-c321e6978d6a

---

Tested further and confirmed worked without flattening - my initial test was flawed.